scripts/helpers/createExtensionManifest.mjs

```diff
@@ -12,11 +12,22 @@ governing permissions and limitations under the License.
 
 import getAlloyComponents from "./getAlloyComponents.mjs";
 
+const ORIGINAL_DEFAULT_COMPONENTS = new Set([
```

Comment:
Maybe we should name this just as DEFAULT_COMPONENTS. If we add a new component later and we want to enable it by default I think we just need to add it to this Set.